themes/soe_basic/soe_basic.theme

```diff
@@ -22,6 +22,17 @@ function soe_basic_theme_suggestions_block_alter(array &$suggestions, array $var
   }
 }
 
+/**
+ * Implements hook_preprocess_page_title_banner().
+ */
+function soe_basic_preprocess_page_title_banner(&$variables) {
```

Comment:
@imonroe am I adding the behaviors incorrectly? I did some debugging and it seems like it doesn't see the `stanford_page_title_banner` on the basic page.

Could it be because the `stanford_page_title_banner` is nested under the `su_page_banner` entity ref field?  🤔